packages/api/webpack.config.js

```diff
@@ -1,26 +1,19 @@
-const path = require('path');
-
-let mode = 'development';
-let devtool = 'inline-source-map';
+'use strict';
 
-if(process.env.PRODUCTION) {
-  mode = 'production',
-  devtool = false  
-}
+const path = require('path');
 
 module.exports = {
-  mode,
+  mode: process.env.PRODUCTION ? 'production' : 'development',
   entry: './index.js',
   output: {
     libraryTarget: 'commonjs2',
     filename: 'index.js',
     path: path.resolve(__dirname, 'dist')
   },
   externals: [
-    'aws-sdk',
```

Comment:
No longer rely on the version of `aws-sdk` that is included in the Lambda runtime.